src/index.html

```diff
@@ -10,23 +10,39 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <section>
```

Comment:
Consider using semantic HTML tags like `section` or `article` to improve SEO and accessibility. This will help search engines and assistive technologies better understand the structure of your content.